deel-2-flexbox-layout/opdracht-3/styles/styles.css

```diff
@@ -1 +1,53 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+/*Schrijf hier jouw CSS*/
+body {
+    height: 953px;
+
+    display: flex;
+    flex-direction: row;
+}
+
+/* ############niveau 2################*/
+nav {
+    flex-basis: 20%;
```

Comment:
viewport width would be better. Percentages are often based on parent containers, viewport units are always consistent with the viewport. If you have a percentage on a direct child of body then they're already off, since body has a default margin.